### DISCUSSION TITLE
Send invitation to users for a poll

### DISCUSSION THREAD

**Discussion by Apokalypt** [lukevella/rallly#334] [Ideas]
Currently, to share a poll it is necessary to copy the link and share it with the people involved. It would be interesting to be able to indicate email addresses to which to send the participation link directly.
What would be even better would be to be able to create a sort of group of email addresses (e.g.: Developer which groups the list of email addresses of the developers, Project Manager which groups the emails of the project managers, ....). When creating a poll, you could select (or not) groups and add emails by hand to notify them of a new poll available.

**Comment by lukevella** [comment]
This is a planned feature but will most likely be available for paying users of the cloud-hosted instance (free on self-hosted). The reason being sending emails isn't free or cheap and I'm hoping power users will be willing to pay for this functionality to help support the development of this project. Curious to hear how others feel about this.

**Reply by Apokalypt** [reply]
For one of my projects, I had to set up an email system. For this I used the Gmail API which is free with a limit of 1,000,000,000 emails per day.
For the purposes of my project, this was fine and may be the case for this project (I have no idea how many users currently). So this feature could remain partially free, but to avoid exceeding the quotas there can be a limit, 10 for those who don't pay and infinite for those who pay.

**Reply by akarikuu** [reply]
Sending out the email is not even the most important thing here imho. But preselecting email addresses (people) from which I expect an answer would give also the possibility in rally to see who already gave a feedback and from whom it is missing. Maybe linked to this discussion?

https://github.com/lukevella/rallly/discussions/201
